src/kakarot/accounts/library.cairo

```diff
@@ -22,7 +22,7 @@ from kakarot.constants import Constants, account_proxy_class_hash
 from kakarot.interfaces.interfaces import IAccount
 
 @storage_var
-func nonce() -> (nonce: felt) {
+func is_evm_address_deployed_(evm_address: felt) -> (res: felt) {
```

Comment:
Added a mapping for existing deployed evm accounts

Reply:
I don't think that we want this, it's like putting back the account registry. At least I'd like to see this in a dedicated PR, then for the same cost I'd have the registry instead, which would give us more flexibility in terms of interoperability